appsdk/sdk.go

```diff
@@ -279,10 +359,31 @@ func (sdk *AppFunctionsSDK) listenForConfigChanges() {
 				return
 			}
 
-			sdk.config.Writable = *actual
+			previousLogLevel := sdk.config.Writable.LogLevel
 
-			sdk.LoggingClient.Info("Writeable configuration has been updated from Registry")
+			sdk.config.Writable = *actual
 			sdk.LoggingClient.SetLogLevel(sdk.config.Writable.LogLevel)
+			sdk.LoggingClient.Info("Writable configuration has been updated from Registry")
+
+			if previousLogLevel != sdk.config.Writable.LogLevel {
```

Comment:
Need to explain why only the log level can be update, i.e. consul sends update per setting.